index.js

```diff
@@ -50,4 +50,12 @@ types.forEach(function(type) {
   normalize[type] = normalize.bind(null, type);
 });
 
+normalize.date = function(value) {
+  function dateOrTimestamp(value) {
```

Comment:
this can be hoisted and then you can use `normalize.date = normalize.bind(null, dateOrTimestamp)`